app/js/components/accordion/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../../../css/accordion.css';
+
+class Accordion extends React.Component {
+  state = {
+    isVisible: this.props.open === undefined ? false : this.props.open,
+  };
+
+  render() {
+    return (
+      <div className={`accordion ${this.props.border ? 'border' : ''}`}>
+        <div className="header">
+          <a
+            role="button"
+            tabIndex={0}
+            onClick={() => {
+              this.setState(() => ({ isVisible: !this.state.isVisible }));
+            }}>
+            <span>
+              {
+                this.state.isVisible ?
+                  <i className="toggle-icon icon-caret-up small rotate180" /> :
+                  <i className="toggle-icon icon-caret-up small rotate90" />
+              }
+            </span>
+            {this.props.title}
+          </a>
+        </div>
+        <div className={`content ${!this.state.isVisible ? 'close' : 'open'}`}>
+          {this.props.children}
+        </div>
+      </div>
+    );
+  }
+}
+
+
+Accordion.propTypes = {
+  border: PropTypes.bool,
+  children: PropTypes.shape({}).isRequired,
```

Comment:
When using `PropTypes.shape`, you should actually define the shape of the expected object.
Here's an example from the docs:

```// An object taking on a particular shape
  optionalObjectWithShape: PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  })

Reply:
Thanks Segun. I will work on this